### PR TITLE
Implemented learn reminder

### PR DESCRIPTION
**Pull request by LockedThread**
This is to remind lemon and kiwi to learn how to code every hour.

### CODE REVIEW

File: jobs/learn_reminder.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const users = ['864311867765686332', '653079222088106005']
+// Kiwi & Lemon
+const LearnReminder = {
+  cronPattern: '0 * * * *', // every hour at minute 0
+  channel: '💬｜general',
+  async execute(channel) {
+    for (let user of users) {
```
Comment (vijaystroup) [inline jobs/learn_reminder.js]: no braces for one line

File: jobs/learn_reminder.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const users = ['864311867765686332', '653079222088106005']
+// Kiwi & Lemon
```
Comment (vijaystroup) [inline jobs/learn_reminder.js]: comment above line 1

**Review by vijaystroup — CHANGES_REQUESTED** [review]

File: jobs/learn_reminder.js (hunk 1 above)
Comment (vijaystroup) [reply jobs/learn_reminder.js]: also you can do `const user of users` instead of `let`

**Review by vijaystroup — CHANGES_REQUESTED** [review]

**Comment by vijaystroup** [comment]
Change filename to `jobs/learnReminder.js` instead of `jobs/learn_reminder.js`

**Comment by LockedThread** [comment]
@VijayStroup 

> Change filename to `jobs/learnReminder.js` instead of `jobs/learn_reminder.js`

Done

**Review by vijaystroup — APPROVED** [review]

**Pull request merged**